### ISSUE TITLE
chore: Refactor segmentio references to aws-okta

### ISSUE THREAD

**Issue by timoguin** [aws-okta/aws-okta#3]
## What

References to `segmentio` in the project need to be replaced with `aws-okta`.

## Why

Two reasons: broken build pipelines and Golang package names.

The CircleCI pipelines are broken because it's trying to publish packages to the [segmentio GH org](https://github.com/segmentio). I've already authorized CircleCI for access to this org, and builds succeed all the way up until the release publishing stages.

Any contributors to the project need to be able to type `go get -u github.com/aws-okta/aws-okta`. Currently, local development requires cloning this repo into `src/github.com/segmentio/aws-okta`

## How

Basically a find-and-replace.

Fixing the build should be as simple as updating the default name in the Makefile and/or adding a variable in the CircleCI project.

Package references all need to be renamed to `aws-okta`.
